src/app/dashboard/layout.tsx

```diff
@@ -32,15 +32,13 @@ export default function DashboardLayout({
       <LeftSidebar />
       <div className="mt-20 ml-20 p-3 pb-7">
         <ErrorBoundary
-          fallbackRender={
-            ({ error }) => {
-              if (!(error instanceof MaffinError)) {
-                error = new MaffinError(error.message, 'UNKNOWN');
-              }
-
-              return error.show();
+          FallbackComponent={ErrorComponent}
+          onError={(error) => {
+            if (!(error instanceof MaffinError)) {
+              error = new MaffinError(error.message, 'UNKNOWN');
             }
-          }
+            (error as MaffinError).show();
+          }}
```

Comment:
So when there is an error triggered, navigation stops working. This is because the state of ErrorBoundary still has an error and thus, keeps rendering the fallback component. I... tried to fix this but haven't managed to :/